kube/overlays/stable/kustomization.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+bases:
+  - ../../resources
+
+images:
+  - name: airbyte/seed
+    newTag: 0.7.2-alpha
```

Comment:
is there any reasonable way to read this from .env? not in love with the fact that there are multiple places we will need to change version. hundo p i will forget.

Reply:
I don't think there's a way to do this actually in kustomize. You can inject ConfigMaps from env files but you can't use values of config maps in something like this. I can add a test to the build that verifies that this matches though.